events/app_watcher.go

```diff
@@ -101,7 +140,16 @@ func (m *AppWatcher) processContainerMetric(metric *sonde_events.ContainerMetric
 	index := metric.GetInstanceIndex()
 	if int(index) < len(m.MetricsForInstance) {
 		instance := m.MetricsForInstance[index]
-		instance.Cpu.Set(metric.GetCpuPercentage())
+
+		cpuPercentage := int(metric.GetCpuPercentage())
```

Comment:
Why did we extract this variable? It seems to add nothing and doesn’t fit the pattern below for diskBytes